build_scripts/build_runner.py

```diff
@@ -723,7 +723,7 @@ def main():
                         choices=['linux', 'embedded', 'pre_si', 'windows'],
                         help='Type of product')
     parser.add_argument('-e', "--build-event", default='commit',
-                        choices=['pre_commit', 'commit', 'nightly', 'weekly'],
+                        choices=['pre_commit', 'commit', 'nightly', 'weekly', 'open_source'],
```

Comment:
not here. You could just change 'pre_si' to 'open_source' in 'product_type' argument above.